### ISSUE TITLE
Calorie definition error

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#121]
Posted on behalf of a REFPROP user:

My goal is to use REFPROP with the cal to Joule conversion of 4.1868 instead of the default value of 4.184. Certain other parameter values are associated with each of these two options, such as:

![image](https://user-images.githubusercontent.com/11966765/53767718-d8cdee80-3e93-11e9-975e-1f12810d2e27.png)

In the REFPROP.XLA VBA code the following lines are within an IF/ENDIF section.
```
hIn = "Flags"
'Use the following line to change the Cal to J conversion factor from 4.184 to 4.1868 J/mol.
'hOut = "CALORIE": Call REFPROPdll(hFld, hIn, hOut, 0&, 0&, 1&, 0#, 0#, z(1), Output(1), hUnits, iUnits, x(1), y(1), x3(1), q, ierr, herr, 10000&, 255&, 255&, 255&, 255&)
```
I have not been able to get the option to work as expected when the ‘ mark is removed from in front of hOut=.
I was able to get it to work once, ``=REFPROP("UNITCONV","E","cal","J",1,0)``  yielded 4.1868.
However, EXCEL reverted back to using the default value of 4.184 after a few other cells were added that called REFPROP for enthalpy, entropy, etc., but before I closed EXCEL. The code still showed that the option to change the default value was active. I understand this code change disappears if EXCEL is closed.

Things I tried:
1.	I uninstalled REFPROP and reinstalled it in a folder called C: REFPROP to get rid of any issue of ghost directories.
a.	I removed all files in ghost directory
b.	I experimented and was not able to get consistent results.
2.	Again, I uninstalled REFPROP, 
a.	reinstalled in C:REFPROP, 
b.	opened REFPROP.XLS and deleted the ‘ mark in front of hOUT=. 
c.	Then saved the file as REFPROP.XLA and closed XCEL. 
d.	Re-opened EXCEL and looked at the REFPROP.XLA code. The hOut= line for calorie conversion was active.
e.	I experimented and was not able to get consistent results.
3.	Again I uninstalled REFPROP and re-installed in C:REFPROP.
a.	I moved the hOUT= statement outside of the IF/ENDIF section and got inconsistent results. It worked at first and then reverted back to the default value after some other cells were added that used REFPROP.
4.	Again I uninstalled REFPROP and re-installed in C:REFPROP without any code changes.
5.	I now assume the default value of 4.184  is in use and proved it shows up by using UNITCONV to change 1 cal to 4.184 Joules.  (  =REFPROP("UNITCONV","E","cal","J",1,0)  )
6.	To move forward, I use a multiplier factor for enthalpy, entropy, and specific heats to adjust the ratio of metric unit values to English unit values so that it is 2.326 kJ/kg = 1 BTU/lbm which is consistent with 4.1868. At least I know this will be consistent and can be altered by changing the cell that holds the multiplier factor. It is not an elegant solution, but I can move forward with work.
Perhaps I need more instruction what to do and what to expect. A few questions:
1.	Does changing the cal to J conversion factor only affect non SI units?
2.	Does the conversion factor get reset by any means other than the code line above?
3.	It appears that the IF/ENDIF code is only activated if INIT is not = 999. Does INIT get reset anywhere else such as when fluids are changed?
4.	Does using two fluids on the same EXCEL sheet for “parallel” calculations in different unit systems confuse EXCEL?
5.	Can the XLA file only be changed by altering the .XLS file code and resaving/replacing as an .XLA file?

I am using EXCEL 16 in Windows 10.

Additional comment:
``=REFPROP("UNITCONV","E","BTU","kJ",1,0)`` yields 1.054350264 which is correct if caltoJ is 4.184, i.e., the default value.
However, if calculating ``=REFPROP("E","methane","TD","E",100,0.14773)`` yields 333.7973 which is BTU/LBM (internal energy).
Using "E" as a code for both "internal energy" (as in H=U+PV) and energy  (as in BTU or J) was a bit confusing.

**Comment by ianhbell** [comment]
See also #17

**Comment by ianhbell** [comment]
One problem with the use of Excel is that the order of calls is indeterminate.  Therefore, tracking down bugs in the Excel interface can be a challenge.  This is different than calling the DLL in a deterministic call order from Python, C++, MATLAB, etc.  Nonetheless, this seems like a bug.

**Comment by EricLemmon** [comment]
This bug has been fixed now and the value of 4.1868 remains in effect once the option to use this value has been set, the XLA file saved, and Excel restarted.
